public/js/services.js

```diff
@@ -1,5 +1,5 @@
-angular.module('starter.services', [])
-  .factory("api", function($q, $http) {
+/* @ngInject */
```

Comment:
The dependency injection in Angular breaks with minification unless a special syntax is used, so the above factory would have to be written as:
```
angular.module('starter.services', [])
  .factory("api", ['$q', '$http', function($q, $http) { /* Factory Code */ }]);
```
This often gets too tedious so, usually we can use some sort of gulp task to automate it. We may want to create a task for this, depending on where being able to minify lies on our priority.